### PR TITLE
Fmsound

### PR DESCRIPTION
**Pull request by rabe-soft**
This PR significantly improves the sound quality of the FM instruments. This applies in particular to the “Drums”. The corresponding standard frequencies have also been adjusted. The standard UI sounds (Notify, Warning ..., Library/User/User/userUtils.asm) have been adjusted so that they sound almost exactly the same as before. 
In addition, a C version of the soundFmt.def file has been added.

### CODE REVIEW

File: Installed/Driver/Sound/SoundBlaster/Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #
 # If you edit it, you will lose your changes, should it be regenerated.
 #
-GEODE           = sblaster
```
Comment (bluewaysw) [inline Installed/Driver/Sound/SoundBlaster/Makefile:6]: Changing the geode name is likely not a good idea here? Could we revert this change?

**Review by bluewaysw — CHANGES_REQUESTED** [review]
Please keep geode name for existing components unchanged.

**Comment by rabe-soft** [comment]
I am also surprised about this because i have only made a new mkmf / pmake depend. I will have a look to see if there is any problem.

**Comment by rabe-soft** [comment]
There is no problem about this
`GEODE           = sblaster` in the makefile produces
grafik
Then typing mkmf. The makefile now contains `GEODE           = soundblaster` and produces the same files:
grafik

I think, the GEODE statement simply contains the name of the code folder (I have tested this). In this particular case, the geode name is specified in the local.mk file
grafik

**Because every new mkmf in the future will generate the statement “GEODE = soundblaster”, my recommendation would be to accept this now.**

Off topic: The 8-character geo name is probably one of the reasons why the EC target has no sound.

**Review by bluewaysw — APPROVED** [review]

**Pull request merged**